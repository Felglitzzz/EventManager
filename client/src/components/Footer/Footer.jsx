import React from 'react';
/**
 * Class Footer
 */
export default class Footer extends React.Component {
  /**
   * @returns {react} Footer component
   */
  render() {
    return (
      <div>
        <footer className="bg-dark mt-5 h-15 pb-3">
          <div className="container pt-5">
            <ul className="pl-0">
              <li className="d-flex justify-content-center">
                <a className="text-light border-2 border-orange icon facebook mr-1"
                  href="#">
                  <i aria-hidden="true"
                    className="fa fa-facebook" />
                </a>
                <a className="text-light border-2 border-orange icon twitter mr-1"
                  href="#">
                  <i aria-hidden="true"
                    className="fa fa-twitter" />
                </a>
                <a className="text-light border-2 border-orange icon ig mr-1"
                  href="#">
                  <i aria-hidden="true"
                    className="fa fa-instagram" />
                </a>
                <a className="text-light border-2 border-orange icon envelope"
                  href="#">
                  <i aria-hidden="true"
                    className="fa fa-envelope" />
                </a>
              </li>
            </ul>
          </div>
          <div className="container d-flex justify-content-center">
            <h5 className="text-center mb-0 text-light">Â© 2018 Eventeria. All rights reserved</h5>
          </div>
        </footer>
      </div>
    );
  }
}
