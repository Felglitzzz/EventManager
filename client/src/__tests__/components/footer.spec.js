import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer/Footer';

describe('Footer Page', () => {
  it('should render the footer page correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
