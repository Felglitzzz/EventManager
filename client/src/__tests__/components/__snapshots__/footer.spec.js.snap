// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Page should render the footer page correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <footer
        className="bg-dark mt-5 h-15 pb-3"
      >
        <div
          className="container pt-5"
        >
          <ul
            className="pl-0"
          >
            <li
              className="d-flex justify-content-center"
            >
              <a
                className="text-light border-2 border-orange icon facebook mr-1"
                href="#"
              >
                <i
                  aria-hidden="true"
                  className="fa fa-facebook"
                />
              </a>
              <a
                className="text-light border-2 border-orange icon twitter mr-1"
                href="#"
              >
                <i
                  aria-hidden="true"
                  className="fa fa-twitter"
                />
              </a>
              <a
                className="text-light border-2 border-orange icon ig mr-1"
                href="#"
              >
                <i
                  aria-hidden="true"
                  className="fa fa-instagram"
                />
              </a>
              <a
                className="text-light border-2 border-orange icon envelope"
                href="#"
              >
                <i
                  aria-hidden="true"
                  className="fa fa-envelope"
                />
              </a>
            </li>
          </ul>
        </div>
        <div
          className="container d-flex justify-content-center"
        >
          <h5
            className="text-center mb-0 text-light"
          >
            © 2018 Eventeria. All rights reserved
          </h5>
        </div>
      </footer>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="container pt-5"
          >
            <ul
              className="pl-0"
            >
              <li
                className="d-flex justify-content-center"
              >
                <a
                  className="text-light border-2 border-orange icon facebook mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-facebook"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon twitter mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-twitter"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon ig mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-instagram"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon envelope"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-envelope"
                  />
                </a>
              </li>
            </ul>
          </div>,
          <div
            className="container d-flex justify-content-center"
          >
            <h5
              className="text-center mb-0 text-light"
            >
              © 2018 Eventeria. All rights reserved
            </h5>
          </div>,
        ],
        "className": "bg-dark mt-5 h-15 pb-3",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ul
              className="pl-0"
            >
              <li
                className="d-flex justify-content-center"
              >
                <a
                  className="text-light border-2 border-orange icon facebook mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-facebook"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon twitter mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-twitter"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon ig mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-instagram"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon envelope"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-envelope"
                  />
                </a>
              </li>
            </ul>,
            "className": "container pt-5",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <li
                className="d-flex justify-content-center"
              >
                <a
                  className="text-light border-2 border-orange icon facebook mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-facebook"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon twitter mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-twitter"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon ig mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-instagram"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon envelope"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-envelope"
                  />
                </a>
              </li>,
              "className": "pl-0",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <a
                    className="text-light border-2 border-orange icon facebook mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-facebook"
                    />
                  </a>,
                  <a
                    className="text-light border-2 border-orange icon twitter mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-twitter"
                    />
                  </a>,
                  <a
                    className="text-light border-2 border-orange icon ig mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-instagram"
                    />
                  </a>,
                  <a
                    className="text-light border-2 border-orange icon envelope"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-envelope"
                    />
                  </a>,
                ],
                "className": "d-flex justify-content-center",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      aria-hidden="true"
                      className="fa fa-facebook"
                    />,
                    "className": "text-light border-2 border-orange icon facebook mr-1",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "aria-hidden": "true",
                      "className": "fa fa-facebook",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      aria-hidden="true"
                      className="fa fa-twitter"
                    />,
                    "className": "text-light border-2 border-orange icon twitter mr-1",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "aria-hidden": "true",
                      "className": "fa fa-twitter",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      aria-hidden="true"
                      className="fa fa-instagram"
                    />,
                    "className": "text-light border-2 border-orange icon ig mr-1",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "aria-hidden": "true",
                      "className": "fa fa-instagram",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      aria-hidden="true"
                      className="fa fa-envelope"
                    />,
                    "className": "text-light border-2 border-orange icon envelope",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "aria-hidden": "true",
                      "className": "fa fa-envelope",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "a",
                },
              ],
              "type": "li",
            },
            "type": "ul",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h5
              className="text-center mb-0 text-light"
            >
              © 2018 Eventeria. All rights reserved
            </h5>,
            "className": "container d-flex justify-content-center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "© 2018 Eventeria. All rights reserved",
              "className": "text-center mb-0 text-light",
            },
            "ref": null,
            "rendered": "© 2018 Eventeria. All rights reserved",
            "type": "h5",
          },
          "type": "div",
        },
      ],
      "type": "footer",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <footer
          className="bg-dark mt-5 h-15 pb-3"
        >
          <div
            className="container pt-5"
          >
            <ul
              className="pl-0"
            >
              <li
                className="d-flex justify-content-center"
              >
                <a
                  className="text-light border-2 border-orange icon facebook mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-facebook"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon twitter mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-twitter"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon ig mr-1"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-instagram"
                  />
                </a>
                <a
                  className="text-light border-2 border-orange icon envelope"
                  href="#"
                >
                  <i
                    aria-hidden="true"
                    className="fa fa-envelope"
                  />
                </a>
              </li>
            </ul>
          </div>
          <div
            className="container d-flex justify-content-center"
          >
            <h5
              className="text-center mb-0 text-light"
            >
              © 2018 Eventeria. All rights reserved
            </h5>
          </div>
        </footer>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="container pt-5"
            >
              <ul
                className="pl-0"
              >
                <li
                  className="d-flex justify-content-center"
                >
                  <a
                    className="text-light border-2 border-orange icon facebook mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-facebook"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon twitter mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-twitter"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon ig mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-instagram"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon envelope"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-envelope"
                    />
                  </a>
                </li>
              </ul>
            </div>,
            <div
              className="container d-flex justify-content-center"
            >
              <h5
                className="text-center mb-0 text-light"
              >
                © 2018 Eventeria. All rights reserved
              </h5>
            </div>,
          ],
          "className": "bg-dark mt-5 h-15 pb-3",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul
                className="pl-0"
              >
                <li
                  className="d-flex justify-content-center"
                >
                  <a
                    className="text-light border-2 border-orange icon facebook mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-facebook"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon twitter mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-twitter"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon ig mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-instagram"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon envelope"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-envelope"
                    />
                  </a>
                </li>
              </ul>,
              "className": "container pt-5",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <li
                  className="d-flex justify-content-center"
                >
                  <a
                    className="text-light border-2 border-orange icon facebook mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-facebook"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon twitter mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-twitter"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon ig mr-1"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-instagram"
                    />
                  </a>
                  <a
                    className="text-light border-2 border-orange icon envelope"
                    href="#"
                  >
                    <i
                      aria-hidden="true"
                      className="fa fa-envelope"
                    />
                  </a>
                </li>,
                "className": "pl-0",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <a
                      className="text-light border-2 border-orange icon facebook mr-1"
                      href="#"
                    >
                      <i
                        aria-hidden="true"
                        className="fa fa-facebook"
                      />
                    </a>,
                    <a
                      className="text-light border-2 border-orange icon twitter mr-1"
                      href="#"
                    >
                      <i
                        aria-hidden="true"
                        className="fa fa-twitter"
                      />
                    </a>,
                    <a
                      className="text-light border-2 border-orange icon ig mr-1"
                      href="#"
                    >
                      <i
                        aria-hidden="true"
                        className="fa fa-instagram"
                      />
                    </a>,
                    <a
                      className="text-light border-2 border-orange icon envelope"
                      href="#"
                    >
                      <i
                        aria-hidden="true"
                        className="fa fa-envelope"
                      />
                    </a>,
                  ],
                  "className": "d-flex justify-content-center",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <i
                        aria-hidden="true"
                        className="fa fa-facebook"
                      />,
                      "className": "text-light border-2 border-orange icon facebook mr-1",
                      "href": "#",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "aria-hidden": "true",
                        "className": "fa fa-facebook",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    "type": "a",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <i
                        aria-hidden="true"
                        className="fa fa-twitter"
                      />,
                      "className": "text-light border-2 border-orange icon twitter mr-1",
                      "href": "#",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "aria-hidden": "true",
                        "className": "fa fa-twitter",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    "type": "a",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <i
                        aria-hidden="true"
                        className="fa fa-instagram"
                      />,
                      "className": "text-light border-2 border-orange icon ig mr-1",
                      "href": "#",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "aria-hidden": "true",
                        "className": "fa fa-instagram",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    "type": "a",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <i
                        aria-hidden="true"
                        className="fa fa-envelope"
                      />,
                      "className": "text-light border-2 border-orange icon envelope",
                      "href": "#",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "aria-hidden": "true",
                        "className": "fa fa-envelope",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    "type": "a",
                  },
                ],
                "type": "li",
              },
              "type": "ul",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h5
                className="text-center mb-0 text-light"
              >
                © 2018 Eventeria. All rights reserved
              </h5>,
              "className": "container d-flex justify-content-center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "© 2018 Eventeria. All rights reserved",
                "className": "text-center mb-0 text-light",
              },
              "ref": null,
              "rendered": "© 2018 Eventeria. All rights reserved",
              "type": "h5",
            },
            "type": "div",
          },
        ],
        "type": "footer",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
